incident-response-plan.md

```diff
@@ -0,0 +1,153 @@
+| Document findings                 | I        | C                           | A                 | I, C   |
+| Publish and share (if approved)   | I        | R, A                        | C                 | C   |
+
+- 🍿 @Discussion: who should be in the team? 
+- 🍿 @Discussion: what is the right name for the team ("THE TEAM")? 
+- 🍿 @Discussion: what is the right name for the report ("THE REPORT")? 
+- 🍿 @Discussion: Should we publish the learning/findings when possible publickly to help the community?
+
+#### Reporter
+
+This person submits A REPORT to THE TEAM and provides detailed information about the incident.
+
+**Responsibilities**
+
+- Submit A REPORT to THE TEAM.
+
+**Expectations**
+
+- Provide detailed information about the suspected vulnerability.
+- Follow responsible disclosure guidelines (adapted to this context).
+- Cooperate with THE TEAM by providing additional details when needed.
+- Respect security timelines and avoid premature public disclosure.
+
+
+
+#### Coordinator (SRC)
+
+This person acts as the focal point for a specific REPORT and ensures the report follows all responsible disclosure guidelines. The SRC coordinates the remediation process if the situation is confirmed and ensures that THE REPORT follows the process and necessary actions are taken. While the SRC is not necessarily responsible for performing a detailed analysis or remediation.
+
+**Responsibilities**
+
+- Acknowledge receipt of REPORTS within the required timeframe.
+- Orchestrate the embargo and identify the minimum set of individuals involved.
+- Remind everyone involved that they must not notify/involve any other individuals. If someone else needs to be involved, that must go through the Coordinator.
+- Assign one or multiple SMEs.
+- Ensure communication with the reporter and the affected projects throughout the process.
+- Track all THE REPORTS for visibility and reporting.
+
+#### Subject Matter Expert (SME)
+Experts brought in for technical insight, platform liaison work, or domain-specific advice.  
+
+**Responsibilities**:  
+- Provide expert input to help assess impact and options  
+- Advise on mitigation strategies  
+- Help unblock the situations when feasible
+
+### Reporting methods
+
+- 🍿 @Discussion: what is the best option?
+    - Dedicated email alias?  
+    - Secure web form?
+
+
+## Runbook
+
+- 🍿 @Discussion: What is the best approach? Some ideas:
+    1. **REPORT Received**  
+    2. **Assign Coordinator** and consolidate report details  
+    3. **Review** severity and affected projects  
+    4. **Identify SMEs** and brief them  
+    5. **Coordinate** with projects, platforms, or third parties  
+    6. **Document** findings and lessons learned  
+    7. **Publish** partial or full summary if appropriate  
+    8. **Social Media Team** prepare and posts where needed 
+
+## General Response Workflow  
+
+- 🍿 @Discussion: early-stage idea, based on the Runbook:
+
+```mermaid
+flowchart TD
```

Comment:
I think we should decide how communication and work here is co-ordinated. For example a common practice is that when an incident occurs, the incident commander creates a dedicated slack channel to facilitate communication.